WooCommerce/WooCommerceTests/UnitTests.xctestplan

```diff
@@ -16,6 +16,40 @@
         "enabled" : false
       }
     ],
+    "environmentVariableEntries" : [
```

Comment:
Yes! I [knew](https://github.com/woocommerce/woocommerce-ios/pull/7097#discussion_r905959634) there was gonna be some way to inject env variables and pass them at runtime! We just had to dig a little deeper 😄  

So much better! 🎉